collectfast/tests/test_command.py

```diff
@@ -62,6 +62,16 @@ def test_collectfast_disabled(case):
     call_collectstatic()
 
 
+@test
+@override_django_settings(
+    STATICFILES_STORAGE="django.contrib.staticfiles.storage.StaticFilesStorage")
+@override_setting("enabled", True)
```

Comment:
did you mean for this to be False? (though I would say we also need to test this particular config)

Reply:
@pachewise Good find! Thanks for pointing that out! :)

Reply:
(And that config is already tested in `test_basics` btw)